aws/resource_aws_emr_instance_group.go

```diff
@@ -55,6 +55,11 @@ func resourceAwsEMRInstanceGroup() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"bid_price": {
```

Comment:
We should also check the value returned by AWS when reading the resource, i.e. in the `resourceAwsEMRInstanceGroupRead` function, around lines 190. 
This helps Terraform knows if the value changed, for example from a manual edit in the console.